crates/swc_ecma_ast/src/expr.rs

```diff
@@ -471,6 +492,102 @@ impl Take for AssignExpr {
     }
 }
 
+impl<'de> Deserialize<'de> for AssignExpr {
+    fn deserialize<D>(deserializer: D) -> Result<Self, D::Error>
+    where
+        D: Deserializer<'de>,
+    {
+        struct AssignExprVisitor;
+
+        impl<'de> Visitor<'de> for AssignExprVisitor {
+            type Value = AssignExpr;
+
+            fn expecting(&self, formatter: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+                formatter.write_str("struct AssignExpr")
+            }
+
+            fn visit_map<M>(self, mut map: M) -> Result<Self::Value, M::Error>
+            where
+                M: MapAccess<'de>,
+            {
+                let mut span_field: Option<Span> = None;
+                let mut op_field: Option<AssignOp> = None;
+                let mut left_field: Option<PatOrExpr> = None;
+                let mut right_field: Option<Box<Expr>> = None;
+
+                while let Some(key) = map.next_key::<&str>()? {
+                    match key {
+                        "span" => {
+                            if Option::is_some(&span_field) {
```

Comment:
Also, please change this to `span_field.is_some()`

Reply:
Done.